src/executables/util_cs.c

```diff
@@ -5066,6 +5060,12 @@ flashback (UTIL_FUNCTION_ARG * arg)
 
   need_shutdown = true;
 
+  if (!prm_get_integer_value (PRM_ID_SUPPLEMENTAL_LOG))
```

Comment:
Why has it been moved?

Reply:
Oh.. this is unrelated change. it is done in other PR (#3385)
I moved this because parameters are loaded after db_restart ().

Reply:
I will revert it when all the feedbacks are applied. 
Now, I just leave it for a smooth test.